client/src/BuyerTenderList.js

```diff
@@ -138,11 +138,8 @@ const BuyerTenderList = () => {
 								)}
 							</p>
 							<p>
-								<strong>Cost: </strong>£{tender.cost}{" "}
-								<span>
-									<strong>No. Of Bids Received: </strong>
-									{tender.no_of_bids_received}
-								</span>
+								<strong>No. Of Bids Received: </strong>
+								{tender.no_of_bids_received}
```

Comment:
This and the confirmation are two different things. Put them in two different PRs